### DISCUSSION TITLE
Loss function

### DISCUSSION THREAD

**Discussion by Vrushank264** [open-mmlab/mmaction2#1442] [Q&A] [answered]
Where can I see and/or change the loss function being used for action recognition training? Specifically for I3D and SlowFast Networks.

**Comment by dreamerlin** [comment, marked as answer]
https://github.com/open-mmlab/mmaction2/blob/master/mmaction/models/heads/base.py#L50

https://github.com/open-mmlab/mmaction2/tree/master/mmaction/models/losses
